game/gothic.cpp

```diff
@@ -575,6 +578,15 @@ std::string_view Gothic::defaultSave() const {
   return CommandLine::inst().defaultSave();
   }
 
+std::string Gothic::getGameDatName() {
+  if (settingsGetS("FILES","Game") != "") {
```

Comment:
Why if-else? `gameDef` appears to be constant (`=="GOTHIC.DAT"`) for most of it's life-time.
This should be implemented same as `plDef`/`wrldDef`

Reply:
ok, this should be now in line with `plDef` & `wrldDef`